openstack/compute/v2/servers/requests.go

```diff
@@ -182,6 +182,7 @@ type CreateOpts struct {
 
 // ToServerCreateMap assembles a request body based on the contents of a CreateOpts.
 func (opts CreateOpts) ToServerCreateMap() (map[string]interface{}, error) {
+	var serverCreateMapError error
```

Comment:
> 2. introduces a fix unique to gophercloud.

I don't follow: In what way could we create a fix that would be non-unique to Gophercloud?

> 3. feels like it's literally pushing the problem away until later.

I agree that 3. is weak, but it would eliminate the need for `serverCreateMapError`.

Of course the most straightforward solution would be to not call `servers.ToServerCreateMap` and instead put the server creation logic in `bfv.ToServersCreateMap` (yes, resulting in code duplication...).